docs/proposals/20200804-windows-support.md

```diff
@@ -0,0 +1,271 @@
+By using cloudbase-init, Windows can leverage the existing solutions and stay up to date with the latest changes in CABPK.
+
+#### VHD Creation
+Using cloudbase-init requires the creation of a VHD with the tooling installed on it since it is not 
+provided out of the box by any cloud providers.  To address this we will provide packer scripts as part of 
+the [image-builder project](https://github.com/kubernetes-sigs/image-builder) that will pre-install 
+cloudbase init.  It is important to note that while scripts can be provided to build a  VHD.  All VHD’s 
+built will need to adhere to [Windows licensing requirements](https://www.microsoft.com/en-us/licensing/product-licensing/windows-server).
+
+AKS-Engine has an example implementation for using packer and scripts to do VHD configuration: https://github.com/Azure/aks-engine/blob/master/vhd/packer/windows-vhd-builder.json
+
+Although the Linux implementation in image-builder uses Ansible for configuration, Windows will not share 
+the configuration. Therefore, Ansible is not a requirement for Windows image builder implementation.  
+The main requirement for VHD building is that the build can be started from a Linux or Windows machine.
+
+#### Kubelet and other component configuration
+Due to the lack of privileged containers in Windows, a combination of `PreKubeadmCommands`/`PostKubeadmCommands` 
+scripts and wins.exe can be used to configure the nodes. Wins.exe is currently provided as a way to  bootstrap nodes along with kubeadm in the [Kubernetes documentation for adding Windows nodes](https://kubernetes.io/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes/).  
+The components from the [preparenode script](https://kubernetes.io/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes/#joining-a-windows-worker-node) can be used during VHD creation.
+
+In the future with the introduction of the [Privileged Containers for Windows containers](https://docs.google.com/document/d/12EUtMdWFxhTCfFrqhlBGWV70MkZZPOgxw0X-LTR0VAo/edit#) we can use Privileged containers in place of wins.exe enabled containers. 
+As the privileged container support is still being discussed, the replacement of the `PreKubeadmCommands`/`PostKubeadmCommands` kubeadm scripts might be possible but will need to be addressed at a later point in time. 
+
+The plan during Alpha implementation is that each of the infrastructure providers will provide their own `PreKubeadmCommands`/`PostKubeadmCommands` scripts. 
+For Beta we should be able to identify common overlapping features that can be added directly to the kubeadm cluster provisioning 
+(CABPK) in a similar way that the [kubeadm-bootstrap-script.sh](https://github.com/kubernetes-sigs/cluster-api/blob/master/bootstrap/kubeadm/internal/cloudinit/kubeadm-bootstrap-script.sh) is done today.
+
+#### netbios names
+Netbios name resolution concerns were addressed in https://github.com/kubernetes-sigs/cluster-api/issues/2217#issuecomment-598999436
```

Comment:
Can you sumamrize outcome here so people skimming this doc don't need to follow the link?